app/routes.py

```diff
@@ -1,9 +1,23 @@
 # View functions
-from flask import render_template, url_for
```

Comment:
I need `url_for` for 'back' button in html templates in the error page. Please, don't delete it.